### PR TITLE
[4.2] add note about max_input_vars

### PR DESCRIPTION
**Pull request by fritzmg**
Added note for https://github.com/contao/core/issues/7938. French version is missing though.

### CODE REVIEW

File: manual/en/01-installation/configuring-the-live-server.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ You also need several PHP extensions to get the full functionality.
 | mbstring      | multi-byte character handling | no        |
 | mcrypt        | data encryption               | no        |
 
+Depending on your installed extensions, it may be necessary to increase the
+PHP variable `max_input_vars` (e.g. `2000` or higher). Otherwise long backend
```
Comment (lionel-m) [inline manual/en/01-installation/configuring-the-live-server.md]: backend -> back end

**Comment by lionel-m** [comment]
Thanks ! The French translation:

> En fonction de vos extensions installées, il peut être nécessaire d'augmenter la valeur de la variable PHP `max_input_vars` (par ex. `2000` ou supérieur). Sinon, les longs formulaires du back office ne peuvent pas être soumis entièrement.

**Comment by lionel-m** [comment]
I think it's better to place this paragraph as a note, because it is not directly related to the Contao core but if an extension displays a too long form. Correct?

So with `> **Info**` at the beginning of your paragraph. [Example](https://docs.contao.org/books/manual/4.2/en/02-administration-area/index.html)

**Comment by fritzmg** [comment]
> I think it's better to place this paragraph as a note, because it is not directly related to the Contao core but if an extension displays a too long form. Correct?
> 
> So with > **Info** at the beginning of your paragraph. Example

Agreed. I'll update soon.

However, I would like to point out that you can run into this problem without extensions as well - it's just very unlikely (you would need to create a lot of news archives, FAQ archives, calendars etc. for that, around 255 entries in total I think).

**Comment by fritzmg** [comment]
I used `> **Note**` now. Or is `> **Info**` more fitting? I don't know ;)

**Comment by lionel-m** [comment]
> I used > **Note** now. Or is > **Info** more fitting? I don't know ;)

As you wish :)

File: manual/fr/01-installation/configuration-du-serveur-en-ligne.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ toutes les fonctionnalités.
 | mbstring      | gestion des caractères multi-octets | non         |
 | mcrypt        | cryptage des données                | non         |
 
+> **Note** En fonction de vos extensions installées, il peut être nécessaire d'augmenter la valeur de la variable PHP max_input_vars (par ex. 2000 ou supérieur). Sinon, les longs formulaires du back office ne peuvent pas être soumis entièrement.
```
Comment (lionel-m) [inline manual/fr/01-installation/configuration-du-serveur-en-ligne.md:34]: Could you add the backtick quotes (`). They were no longer visible in the comment. Thx

**Comment by aschempp** [comment]
Thanks for the PR! It is not necessary to create a pull request for each version of the manual. We simply merge changes from 3.5 into later releases. So can you please update issue #390 to contain all changes mentioned here? Then I can merge that into all manuals.

**Comment by fritzmg** [comment]
done, I've updated #390 
I guess I can close this PR and delete the branch?

**Comment by aschempp** [comment]
Yes, but it looks like you forgot the latest commend from @lionel-m :)

**Comment by fritzmg** [comment]
right, done :)

**Pull request closed without merging**